Update configure-delegate-proxy-settings.md (#726)
harness/developer-hub: docs/platform/2_Delegates/configure-delegates/configure-delegate-proxy-settings.md

```
kubectl -n default get service kubernetes -o json | jq -r '.spec.clusterIP'
```

:::info

> Harness also support mTLS functionality on a case to case basis, please contact Harness support to enable it.

Reviewer: ```suggestion
Harness supports mTLS authentication on a case-by-case basis. Contact Harness Support to enable it.
```
